### DISCUSSION TITLE
Attach pytest logs to steps

### DISCUSSION THREAD

**Discussion by boris-becker-p** [allure-framework/allure2#1435] [Ideas & Feedback]
It will be great to have the option to split huge pytest logs to different allure steps

**Comment by Avaddon-tfa** [comment]
Hello there! 

Allure Framework is flexible both in data sources available and in reporting configuration. 
Please, give us an example file with the specific expectations on which parts you'd like to split and where. 

We'll take a look and will try to make a guide for that.

**Comment by imool5** [comment]
Hi @Avaddon-tfa I have similiar problem im logging logs into file with logger configuration like this (in conftest)

``` 
def configure_logger(config):
    logging.getLogger('faker').setLevel(logging.INFO)
    timestamp = datetime.strftime(datetime.now(), '%Y-%m-%d_%H-%M-%S')
    path = f'logs/automatic_test_{timestamp}.log'
    path = os.path.join(os.path.abspath(f"{__file__}/../../"), path)
    logging.basicConfig(
        format=config.getini('log_file_format'),
        level=config.getini('log_file_level'),
        filename=path
    )
```

im using pytest

logs in console and in file looks like this:

![image](https://github.com/allure-framework/allure2/assets/34322724/b51c0d11-89a0-4f84-b545-d3e965e2aa7f)

but in allure report

![image](https://github.com/allure-framework/allure2/assets/34322724/97524a9a-b90a-4012-9b89-1b68a7a43248)

it doesnt look like configured one and it hurts me to not have timestamps on report.

Could I also additionally add logs (attach(?)) into test cases from external source (tested application for example somehow) ?
